packages/dts-test/defineComponent.test-d.tsx

```diff
@@ -1428,6 +1494,7 @@ declare const MyButton: DefineComponent<
   ComponentOptionsMixin,
   EmitsOptions,
   string,
+  {},
```

Comment:
Missed this part when I was reviewing - we cannot actually change the type arguments order of `DefineComponent` because it's a publicly-exported type and used in library type definitions.